composer.json

```diff
@@ -20,10 +19,9 @@
     "install-codestandards": [
       "Dealerdirect\\Composer\\Plugin\\Installers\\PHPCodeSniffer\\Plugin::run"
     ],
-    "phpcs-dev": "\"vendor/bin/phpcs\"",
-    "run-phpcs": [
-      "@phpcs-dev"
-    ],
+    "phpcs-i": "@php ./vendor/squizlabs/php_codesniffer/bin/phpcs -i",
+    "phpcs-dev": "@php ./vendor/squizlabs/php_codesniffer/bin/phpcs",
+    "phpcs-fix": "@php ./vendor/squizlabs/php_codesniffer/bin/phpcbf",
```

Comment:
These two commands should use `vendor/bin/phpcs` and `vendor/bin/phpcbf` respectively.